libsolidity/parsing/Parser.h

```diff
@@ -175,6 +175,9 @@ class Parser: public langutil::ParserBase
 		bool empty() const;
 	};
 
+	/// Returns the next AST node ID
+	int64_t nextID() { return ++m_currentNodeID; }
```

Comment:
Is there any special to pre-increment? (e.g. to not have 0 as an ID). If so, please leave a comment.

Reply:
I am not really sure, I just know that the previous node id dispencer used the same logic

Reply:
I don't know the answer to that, only that the previous logic did the same